examples/plot_sleep_staging_usleep.py

```diff
@@ -0,0 +1,348 @@
+"""
+Sleep staging on the Sleep Physionet dataset using U-Sleep network
+==================================================================
+
+.. note::
+    Please take a look at the simpler sleep staging example
+    `here <https://braindecode.org/auto_examples/plot_sleep_staging.html>`_
```

Comment:
do not do it like this. All sphinx gallery examples have an anchor you can use for a link.
Here this would point to stable release although it should point to dev version.

Anchor is called sphx_glr_auto_examples_plot_sleep_staging.py so I think it could be:
```suggestion
    :ref:`here <sphx_glr_auto_examples_plot_sleep_staging.py>`
```